federator/tsconfig.json

```diff
@@ -0,0 +1,12 @@
+{
+  "compilerOptions": {
+    "outDir": "./built",
+    "allowJs": true,
+    "target": "es5",
```

Comment:
why not es6? sorry just curious haha

Reply:
I had some problems using es6 while we have to handle interoperability between `.js` and `.ts` files, so to fix it I set to es5 for now

Reply:
totally reasonable